databricks/koalas/base.py

```diff
@@ -212,8 +214,14 @@ def rtruediv(left, right):
 
     def __floordiv__(self, other):
         def floordiv(left, right):
-            return F.when(F.lit(right == 0), F.lit(np.inf).__div__(left)).otherwise(
-                F.when(F.lit(right) == np.nan, np.nan).otherwise(F.floor(left.__div__(right)))
+            return F.when(F.lit(right is np.nan), np.nan).otherwise(
```

Comment:
Let's just add a docstring to document the symantic differences between PySpark and pandas in the docstring at `__floordiv__` and `__truediv__`.

Reply:
@HyukjinKwon Sounds good. I'll add it, thanks!